stellar-captive-core/debian/stellar-captive-core.preinst

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -e
+
+if [ -f /lib/systemd/system/stellar-core.service ]; then
+  /bin/systemctl disable stellar-core
+fi
+
+if [ -f /lib/systemd/system/stellar-horizon.service ]; then
+  /bin/systemctl stop stlelar-horizon
```

Comment:
I doubt this is a problem but do we need to stop horizon here? I think normally config changes followed by restart in postinst is enough

Reply:
The issue is that this virtual package is intended for use both as a testnet deployment and a pubnet deployment, and the user has to pick which configuration they're going to use before they can bootstrap the node.  Because of that, I don't think it makes sense to automatically start horizon.